Move LlmConfig to extension/llm

jackzhxng opened the pull request:
Moves `LlmConfig` from `examples/models/llama/config` to `extension/llm/export/config`

digantdesai reviewed (commented):
LGTM, this is just a mechanical move? i.e. no functional change otherwise?

Also I assume moving this in extensions/llm is been through a review of sorts?

jackzhxng committed:
Move LlmConfig to extension/llm (#11872)

Summary:
Moves `LlmConfig` from `examples/models/llama/config` to `extension/llm/export/config`

Pull Request resolved: https://github.com/pytorch/executorch/pull/11872

Differential Revision: D77338897

Pulled By: jackzhxng
